src/htmlContent/copy-paste-dialog.html

```diff
@@ -0,0 +1,51 @@
+<div id="myModal" class="modal">
```

Comment:
![current](https://cloud.githubusercontent.com/assets/11540135/24365854/4d192f8e-12e5-11e7-9e82-bc1aefbcee71.png)
@flukeout it currently looks like this.